.github/workflows/dev-ci.yml

```diff
@@ -28,9 +28,8 @@ jobs:
       - name: Setup GraalVM
         uses: graalvm/setup-graalvm@v1
         with:
-          version: '22.2.0'
-          java-version: '11'
-          components: 'native-image'
+          java-version: '17'
```

Comment:
we can go with java 17 even if powsybl core is not in java 17 ?

Reply:
Yes, in fact whatever the version of Java we choose in pypowsybl, it won't had any impact on anything as java code is at the end compiler to native binary code. Oracle GraalVM is available only for 17 and 20. We could also go directly to 20 with any impact.